install/kubernetes/helm/istio/values.yaml

```diff
@@ -136,7 +136,7 @@ global:
     resources:
       requests:
         cpu: 10m
-      #  memory: 128Mi
+      #   memory: 32Mi
```

Comment:
how about we enable this only when auto scaling is enabled??

Reply:
As in tell sidecars that if the workload they are sidecarring is controlled by a memory based hpa... then have a mem request configured, if not no?

Reply:
ok I realized you don't know if the app will be using auto scaling or not at this point during istio configuration/.

Reply:
I would uncomment the memory here too for the sidecar.

Reply:
Done!